extension/src/setup/index.ts

```diff
@@ -314,12 +338,19 @@ export class Setup
       return
     }
 
-    return this.storeSecret(STUDIO_ACCESS_TOKEN_KEY, token)
+    await this.internalCommands.executeCommand(
+      AvailableCommands.CONFIG,
+      cwd,
+      Flag.GLOBAL,
```

Comment:
Should we mention somewhere that the Studio token is saved globally? Or that users with `--system` can't remove their tokens?

Reply:
The docs say this:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/37993418/235245133-abcc9855-59a2-4da7-ac1c-871a4455eb4a.png">

I did consider trying to accommodate "all possibilities" but really we are trying to help beginners with these actions. If the user is competent enough to save a token in their `--system` config then I do not think they'll try to manage it in this way. I feel like even trying to delete `--local` is really going above and beyond + makes the code messy enough.

Reply:
if anyone complains we will revisit